### ISSUE TITLE
Support for more files

### ISSUE THREAD

**Issue by anurag3301** [hugolgst/vimsence#54]
Can you please add support for more filetypes or can you guild me so I might be able to add more and create a pull request for that.
+ .csv (Comma Selected Values)
+ .ino (Arduino)
+ .v (Verilog)
+ .asm (Assembly)
+ .hs (Haskell)

Rest Ill let you know

**Comment by anned20** [comment]
Yes we can. I can make the code change, but could you provide the images for the respective languages? Transparent images are required.

**Comment by anurag3301** [comment]
@anned20 sure keep posting about new extensions and attaching transparent png images. Can you please specify the resolution of the image along with an example?

**Comment by anurag3301** [comment]
I was thinking to use icons from vscode-material-icon. It would save a lot of time for gathering icons. I'll get each svg image and convert it into a transparent png image and post here.
Check out the github repo: https://github.com/PKief/vscode-material-icon-theme/tree/main/icons
